docs/src/css/custom.css

```
@@ -1,33 +1,150 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300..800&family=JetBrains+Mono:wght@400;700&display=swap');
+
+/* ===== Light Mode (default Infima overrides) ===== */
 :root {
-  --ifm-color-primary: #3e0d60;
-  --ifm-color-primary-dark: #360b55;
-  --ifm-color-primary-darker: #2e094a;
-  --ifm-color-primary-darkest: #23073a;
-  --ifm-color-primary-light: #4b1573;
-  --ifm-color-primary-lighter: #561c85;
-  --ifm-color-primary-lightest: #642a95;
+  --ifm-font-family-base: 'Inter', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)